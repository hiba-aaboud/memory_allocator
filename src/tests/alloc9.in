a50000
a3000
a4000
a2000
a1500
f2
f4
a1500
a2500
f6
a3000
f5
f7
a2000
f8
f9
f3
f1
q
