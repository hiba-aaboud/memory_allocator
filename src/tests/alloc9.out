ALLOC at : 8 (50000 byte(s)) -- pool 3
ALLOC at : 50024 (3000 byte(s)) -- pool 3
ALLOC at : 53040 (4000 byte(s)) -- pool 3
ALLOC at : 57056 (2000 byte(s)) -- pool 3
ALLOC at : 59072 (1500 byte(s)) -- pool 3
FREE  at : 50024 -- pool 3
FREE  at : 57056 -- pool 3
ALLOC at : 57056 (1500 byte(s)) -- pool 3
ALLOC at : 50024 (2500 byte(s)) -- pool 3
FREE  at : 57056 -- pool 3
ALLOC at : 60588 (3000 byte(s)) -- pool 3
FREE  at : 59072 -- pool 3
FREE  at : 50024 -- pool 3
ALLOC at : 50024 (2000 byte(s)) -- pool 3
FREE  at : 60588 -- pool 3
FREE  at : 50024 -- pool 3
FREE  at : 53040 -- pool 3
FREE  at : 8 -- pool 3
